java-squid/src/main/java/org/sonar/java/se/ConstraintManager.java

```diff
@@ -96,16 +99,16 @@ public SymbolicValue eval(ProgramState programState, Tree syntaxNode) {
         }
         break;
       }
-      case LOGICAL_COMPLEMENT : {
+      case LOGICAL_COMPLEMENT: {
         UnaryExpressionTree unaryExpressionTree = (UnaryExpressionTree) syntaxNode;
         SymbolicValue val = eval(programState, unaryExpressionTree.expression());
-        if(SymbolicValue.FALSE_LITERAL.equals(val)) {
+        if (SymbolicValue.FALSE_LITERAL.equals(val)) {
           return SymbolicValue.TRUE_LITERAL;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 13. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AIndentationCheck)